samples/DotNet/Azure.Messaging.EventHubs/ManagedIdentityWebApp/SendReceive.aspx.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+// Always add app to IAM roles
+// Don't use on deployment slots but only on root
+namespace ManagedIdentityWebApp
+{
+    using System;
+    using System.Text;
+    using System.Threading;
+    using Azure.Messaging.EventHubs;
+    using Azure.Identity;
+    using System.Threading.Tasks;
+
+    public partial class SendReceive : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        protected async void btnSend_Click(object sender, EventArgs e)
+        {
+            await using (EventHubProducerClient producerClient = new EventHubProducerClient(txtNamespace.Text, txtEventHub.Text, new DefaultAzureCredential()))
+            {
+                // create a batch
+                EventDataBatch eventBatch = await producerClient.CreateBatchAsync();
```

Comment:
nit:  `EventDataBatch` is disposable;  you'll want to apply a `using` scope to it.   Same for the the other uses.